### ISSUE TITLE
Ruins Invalid Data Report - GS46 - N01

### ISSUE THREAD

**Issue by CBurleigh81** [canonn-science/RuinsComboMap#33]
***Please note:*** Tech 6 obelisks have been confirmed to be bugged by Frontier. Mission cannot be completed until Frontier rolls out a fix for that.

## Your CMDR Name:
CMDR DaymienRogue

## Which Ruins Site?
GS046

## Which Obelisk ID?
N01

## What data did you get?
NONE

## What combo did you use?
Casket+Urn

## Other Questions/Comments/Concerns
Another History 3 Obelisk not giving data. Scanned in all game modes. Does not give Hist 3 data nor does ot give wrong combo message.

**Comment by derrickmehaffy** [comment]
Possible reporter already has History 3?

**Comment by derrickmehaffy** [comment]
Confirmed by multiple other commanders this combo works here, marking as resolved as its likely the user already has History 3

**Comment by CBurleigh81** [comment]
dearest CMDRs it was bugged for me somehow. When I submitted the reports on Hist 3 above I had meticulously been keeping track on a spreadsheet every scan i received. I did finally receive His 3 at another site which also confirms I did not have Hist 3 yet. I think it was just some kind of bug only I was experiencing. not 100% sure. Anyways thank you for your time, I just wanted to keep you informed on the progress.
